Sming/Libraries/SDCard/SDCard.h

```diff
@@ -11,10 +11,10 @@ Descr: Low-level SDCard functions
 #include <SmingCore.h>
 #include "SPISoft.h"
 
-void SDCard_begin(uint8 PIN_CARD_SS);
+void SDCard_begin(uint8 PIN_CARD_SS, uint8 byteOrder, uint32 freqLimit);
```

Comment:
Use `uint8_t` rather than the non-standard `uint8`, simililarly `uint32` -> `uint32_t`

Reply:
done